### PR TITLE
Overview page: welcome view for "no-circles" user + UI updates

### LINKED ISSUE #1124: 1.5.4  Implement Overview Page Updates
### Summary

This ticket is to update the current overview page (https://app.coordinape.com/circles) to reflect the new 1.5 app design. There are new design changes to the page, and changes to the default empty state of the page for new users. The scope includes updating the header, styling, and adding buttons. 

- [x] Users belonging to Orgs/ Circles see the new designs
Our current instance needs to reflect the design **below** 
[Overview UI ](https://www.figma.com/file/L4oZwYVUdpgacZtwipGaYe/App-1.5?node-id=4702%3A11975 )

- [x] New users see the Get Started content if they belong to no Orgs / Circles.
New User (not a part of any Orgs / Circles) Welcome Page 
[Overview New User Exp ](https://www.figma.com/file/L4oZwYVUdpgacZtwipGaYe/App-1.5?node-id=6434%3A27983)

If you have any questions about the designs, or things are not clear from the Figma, please reach out in Discord!

### Specification

**Overview Spacing** 
Organizations should be 48px apart. Circles should be 32px apart. 

**Overview Header** 
Add a header to the page with the  "Overview"  the subtitle should be "All your organizations and circle in one place."  in the same div.

**Header Styling** 
font-weight: 600;
font-size: 48px;

**Organizations Logo** 
The organization logo should only be 32x32px. 

**Circle Card Styling** 
The circle name in the card should be **H3 styling.** Epoch Name should be **H3 Style**, and the Date should be H3 Bold. Both should be on the same line instead of stacked. 

**H3 CSS**
font-weight: 600;
font-size: 24px;
line-height: 120%;

**Circle Status** 
The circle status should be in the second row. The styling should be body small. 

**Body Small**
font-weight: 400;
font-size: 13px;
line-height: 130%;

### Assumption 
I assume that the styling referenced above is in our design library. Some of the classes may not reflex the intended design in Figma. Please use Figma styling as a source of truth. If the dev instance doesn't reflect that, please update. 

### After Deployment Checklist:
- [ ] Update "Get Started" on the homepage to link to this page - **this is not part of this ticket**

### Stakeholders
@smombartz

### PR DESCRIPTION
**Pull request by wingfeatherwave**
resolves #1124 

* When a user with no circles clicks "get started" from our public landing page, they will (soon, after UX changes the link) be directed to `/circles`, where they will see a this new view
* UI updates to the "has-circles" view
* Update Org avatar inline uploader
overview pages

### CODE REVIEW

File: src/icons/Torso.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/display-name */
```
Comment (levity) [inline src/icons/Torso.tsx:1]: if this is replacing the one in ui/icons, please remove that one

Comment (levity) [reply src/icons/Torso.tsx:1]: thanks for not adding "Icon" to the end of the filename -- i find that sort of thing redundant. but we should be consistent, so with SettingsIcon also in this PR, let's have it just be one way or the other

File: src/components/OrgLogoUpload/OrgLogoUpload.tsx [hunk 2]
```diff
@@ -3,11 +3,11 @@ import React, { useState } from 'react';
 import { fileToBase64 } from 'lib/base64';
 import { updateOrgLogo } from 'lib/gql/mutations';
 
-import CloseIcon from '@material-ui/icons/Close';
+// import CloseIcon from '@material-ui/icons/Close';
```
Comment (levity) [inline src/components/OrgLogoUpload/OrgLogoUpload.tsx:6]: remove

**Review by levity — CHANGES_REQUESTED** [review]
just a few small things

i feel weird about `$2sm` tbh, but maybe i'll just get used to it

**Comment by levity** [comment]
@ivyquinzel @smombartz please weigh in on this one

File: src/icons/Torso.tsx (hunk 1 above)
Comment (wingfeatherwave) [reply src/icons/Torso.tsx:1]: All of our stitches icons have the `/* eslint-disable react/display-name */` :( What is the proper way to add display name?   I'm not seeing any examples yet

Comment (levity) [reply src/icons/Torso.tsx:1]:
```js
Torso.displayName = 'Torso';
```

**Comment by wingfeatherwave** [comment]
> 

Yea, I'm on the fence about it.  The designs have 16px several places, and the solution would be to use a calc or create a new variable.  Do you have other ideas @levity

**Comment by ivyquinzel** [comment]
is there a vercel link or something I can peak at the front in view with ? @levity @wingfeatherwave

**Comment by ivyquinzel** [comment]
> > re: `2sm`
> 
> Yea, I'm on the fence about it. The designs have 16px several places, and the solution would be to use a calc or create a new variable. and what size is between md and lg, lol? Do you have other ideas @levity
Sizing   Spacing

These are the sizes in the Figma design library. I think we just make an x-med or change how we size to reflect the graphic attached.

**Review by ivyquinzel — APPROVED** [review]

**Comment by wingfeatherwave** [comment]
@ivyquinzel 
> is there a vercel link or something I can peak at the front in view with ? @levity @wingfeatherwave

Yes: https://coordinape-ahj68le8q-coordinape.vercel.app/ 

And it's also behind the button above titled `View deployment`

**Comment by smombartz** [comment]
4, 8, 16, 24, 32, 40 are all sizes we use frequently. I would find the the abstraction more confusing then helpful but thats your call. I think i would do something which takes 8 as base size (xs=4, s=8, m=16, l=24, xl=32, xxl=40) and then from there on maybe its best to have multiples?

**Review by levity — APPROVED** [review]

**Comment by wingfeatherwave** [comment]
> 4, 8, 16, 24, 32, 40 are all sizes we use frequently. I would find the the abstraction more confusing then helpful but thats your call. I think i would do something which takes 8 as base size (xs=4, s=8, m=16, l=24, xl=32, xxl=40) and then from there on maybe its best to have multiples? not sure if this is helpful, let me know if there is a more specific questions you have for the design team.

Padding/Margin refactored here: https://github.com/coordinape/coordinape/pull/1194

**Pull request merged**